### PR TITLE
fix: update metadata

### PR DESCRIPTION
**Pull request by Slesarew**
Update almost all metadata to today morning's state.
Please check carefully.

### CODE REVIEW

File: src/constants/networkMetadata.ts [hunk 1]
```diff
@@ -14,20 +14,25 @@
 // You should have received a copy of the GNU General Public License
 // along with Parity.  If not, see <http://www.gnu.org/licenses/>.
 
+// This file is supposed to have default built-in metadata and should 
+// be upgraded in a timely manner regardless from metadata importing
+// tools.
+// TODO: automate updates
```
Comment (dvdplm) [inline src/constants/networkMetadata.ts:20]: I think it's better to add an issue than a TODO. If you want the todo in the code it should be "TODO: automate updates (issue: … …)"

**Review by dvdplm — CHANGES_REQUESTED** [review]

**Review by TarikGul — APPROVED** [review]
Just as a note for the PR, there are 4 networks that the Metdata changed for:

Westend,
Edgeware
Polkadot
Rococo

LGTM, but I agree with David and would like to see that change as well for the comment.

**Review by s3krit — APPROVED** [review]
LGTM - are you just getting this by calling `state_getMetadata` with `websocat` or similar?

**Comment by Slesarew** [comment]
> LGTM, but I agree with David and would like to see that change as well for the comment.
But it's done, just waiting for David to confirm whether he likes it or not.

**Comment by Slesarew** [comment]
> LGTM - are you just getting this by calling `state_getMetadata` with `websocat` or similar?

yes, these I pulled by manually querying the polkadot.js

**Comment by TarikGul** [comment]
> But it's done, just waiting for David to confirm whether he likes it or not. 

@Slesarew Oh nice, my bad. For some reason I overlooked it.

**Review by dvdplm — APPROVED** [review]

**Pull request merged**